packages/expandable-search-bar/src/expandable-search-bar.ts

```diff
@@ -0,0 +1,261 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  CSSResult,
+  TemplateResult,
+} from 'lit-element';
+
+import magnifyingGlassIcon from './assets/img/magnifying-glass';
+import disclosureTriangleIcon from './assets/img/disclosure';
+import clearResultsIcon from './assets/img/clear-results';
+
+import QuickSearchEntry from './models/quick-search-entry';
+
+import './quick-search';
+
+@customElement('expandable-search-bar')
+export default class ExpandableSearchBar extends LitElement {
```

Comment:
not sure if JSDocs were updated across PRs. I don't see it here...

Reply:
They're there now.